tests/unit/test_wagtail_hooks.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from wagtail.admin import widgets
+from wagtail.core import hooks
+
+from tests.factories.pages import ArticleFactory
+from wagtailtrans.models import TranslatablePage
+from wagtailtrans.wagtail_hooks import edit_in_language_button
+
+
+@pytest.mark.django_db
+class TestWagtailHooks:
+    def setup(self):
+        self.article = ArticleFactory()
+        self.transpage = TranslatablePage()
+
+    @hooks.register('register_page_listing_buttons')
```

Comment:
Why are you defining this here? I think we can just import it right, just like the `edit_in_language_button`, now you're testing code which is defined in the test, seems a bit weird..